libs/api/domains/rsk-company-info/src/lib/graphql/dto/getRskCompanyInfoInput.ts

```diff
@@ -0,0 +1,9 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString } from 'class-validator'
+
+@InputType()
+export class GetRskCompanyInfoInput {
```

Comment:
Both the filename and the GraphQL Input are not following the filenaming conventions of Island.is:
https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions

Prefer `RskCompanyInfoInput` over `Get.*`.
The filename would then be `*.input.ts`.

This goes for the other inputs as well.